File: javascript/chapter3/more_guests.js
let guest_list = ["Melkey", "Johnny Depp", "Linus Trovalds"];

console.log(`Dear ${guest_list[0]}, you are invited to my party!!`);
console.log(`Dear ${guest_list[1]}, you are invited to my party!!`);
console.log(`Dear ${guest_list[2]}, you are invited to my party!!`);

console.log(`\nUnfortunately ${guest_list[2]} can't make it.\n`);

guest_list = ["Melkey", "Johnny Depp", "Rich Harris"];
console.log(`Dear ${guest_list[0]}, you are invited to my party!!`);
console.log(`Dear ${guest_list[1]}, you are invited to my party!!`);
console.log(`Dear ${guest_list[2]}, you are invited to my party!!`);

console.log(`\nI found a bigger table for the guests!!\n`);
guest_list.push("The Primegean");
guest_list.push("Wagslane");

console.log(guest_list);

File: go/chapter3/more_guests.go
package main

import "fmt"

func main() {
	guestList := []string{"Melkey", "Johnny Depp", "Linus Trovalds"}

	fmt.Printf("Dear %s, you are invited to my party!!\n", guestList[0])
	fmt.Printf("Dear %s, you are invited to my party!!\n", guestList[1])
	fmt.Printf("Dear %s, you are invited to my party!!\n", guestList[2])

	fmt.Printf("\nUnfortunately %s can't make it.\n\n", guestList[2])

	guestList = []string{"Melkey", "Johnny Depp", "Rich Harris"}
	fmt.Printf("Dear %s, you are invited to my party!!\n", guestList[0])
	fmt.Printf("Dear %s, you are invited to my party!!\n", guestList[1])
	fmt.Printf("Dear %s, you are invited to my party!!\n", guestList[2])

	fmt.Printf("\nI found a bigger table for the guests!!\n\n")
	guestList = append([]string{"The Primegean"}, guestList...)
	guestList = append(guestList, "Wagslane")

	fmt.Println(guestList)
}
